src/utils.rs

```diff
@@ -19,8 +19,45 @@ use collection::{Collection, CollectionWithId, Id};
 use csv;
 use failure::ResultExt;
 use objects::{AddPrefix, Date};
+use std::fs;
+use std::io::{Read, Write};
 use std::path;
-
+use walkdir::WalkDir;
+use zip;
+
+pub fn zip_to<P: AsRef<path::Path>>(source_path: P, zip_file: P) -> ::Result<()> {
+    let source_path = source_path.as_ref();
+    let file = fs::File::create(zip_file.as_ref())?;
+    let walkdir = WalkDir::new(source_path);
+    let it = walkdir.into_iter();
+
+    let mut zip = zip::ZipWriter::new(file);
+    let options = zip::write::FileOptions::default()
+        .compression_method(zip::CompressionMethod::Bzip2)
+        .unix_permissions(0o755);
```

Comment:
Default permissions should be OK, and 0o644 are better permissions for a zip file.

Reply:
you're right, i'm removing permissions